frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightContent.tsx

```diff
@@ -48,7 +51,20 @@ const PipelineRunDrawerRightContent: React.FC<PipelineRunDrawerRightContentProps
             <Title headingLevel="h2" size="xl">
               {task.name}
             </Title>
-            {task.status?.podName && <Text component="small">{task.status.podName}</Text>}
+            {task.status?.podName && (
+              <div style={{ display: 'inline-flex' }}>
```

Comment:
please use a patternfly component when you can

https://www.patternfly.org/layouts/flex#inline